### PR TITLE
Update files

### PR DESCRIPTION
**Pull request by mwakiogeofrey**
Add .gitignore with workflows and use of linters
Change in index.html and css files

### CODE REVIEW

**Review by RaoAkif — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST :x:

Hi @mwakiogeofrey, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request  :x:

You have submitted a project, that does not contains a suitable title and description of the Pull Request. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form id="survey-form">
+      <label for="name" id="name-label">Name</label><br>
+      <input id="name" type="text" name="name" placeholder="Enter your name" required></input><br><br>
+      <label for="email" id="email-label">Email</label><br>
+      <input id="email" type="email" name="email" pattern="[^ @]*@[^ @]*" placeholder="Enter your email" required></input><br><br>
+      <label for="number" id="number-label">Age(optional)</label><br>
+      <input id="number" type="number" name="number" placeholder="Age" min="18" max="80" required><br><br>
```
Comment (mwakiogeofrey) [inline index.html:14]: - [ ]  You probably forgot to state this as required too

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <select id="dropdown">
+        <option value="none" selected disabled >Select current role</option>
+        <option value="Student"  >student</option>
+        <option value="Full time job">Full time job</option>
+        </select><br><br>
+<label for="name" id="radio-label">Would you recommend freeCodeCamp to a friend?</label><br><br>
+
+<input id="Definitely" type="radio" id="Definitely" name="group1" value="Definitely" >Definitely</input><br>
+<input id="May be" type="radio"  name="group1" value="may be" >May be</input><br>
+<input id="Not sure" type="radio" name="group1" value="Not sure" >Not sure</input>
```
Comment (mwakiogeofrey) [inline index.html:25]: - [ ] You probably forgot to state this as required too

File: style.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#title{
+    text-align: center;
+  }
+  #description{
+    text-align: center;
+  }
+  body{
+    background-image: url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+    
+  }
```
Comment (mwakiogeofrey) [inline style.css:10]: Kindly add more style to the file to make it look more awesome

**Review by mwakiogeofrey — COMMENTED** [review]
good job so far @mwakio
There are some issues that you still need to work on to go to the next project but almost there!

  # required Changes♻
Check the comments under the review.

# Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Pull request merged**